app/index.html

```diff
@@ -152,11 +152,6 @@ <h2 class="text-center" id="temporary-loading-message" style="display: none;">Lo
     <script src="bower_components/file-saver/FileSaver.js"></script>
     <script src="bower_components/hopscotch/dist/js/hopscotch.js"></script>
     <script src="bower_components/origin-web-common/dist/origin-web-common.js"></script>
-    <script src="bower_components/tv4/tv4.js"></script>
-    <script src="bower_components/objectpath/lib/ObjectPath.js"></script>
-    <script src="bower_components/angular-schema-form/dist/schema-form.js"></script>
-    <script src="bower_components/angular-schema-form/dist/bootstrap-decorator.js"></script>
-    <script src="bower_components/angular-schema-form-bootstrap/bootstrap-decorator.min.js"></script>
```

Comment:
Looks like you have some diffs that shouldn't be here. Can you stop grunt serve and try

```
$ git checkout master app/index.html
$ bower install
$ grunt build
```

Then squash your commits?